data/utilities/telephone-infrastructure-and-rural-telcom-boundaries/index.html

```diff
@@ -10,17 +9,17 @@
 categories: []
 date: 2012-05-22 17:58:01 -0600
 Telcom:
-  downloads:
-    - url: https://drive.google.com/drive/folders/0ByStJjVZ7c7mb0hfdDRCaWtlS1U
-      label: Rural Telcom Boundaries
+  hub:
+    name: Utah Rural Telecom Boundaries
+    item_id: be469085c5f747d4b3537317cde88c11
```

Comment:
**question**: is this the correct item id?

Reply:
yes